dashboard/app/models/school.rb

```diff
@@ -382,6 +382,27 @@ def self.seed_from_s3
           }
         end
       end
+
+      CDO.log.info "Seeding 2019-2020 private school data."
+      AWS::S3.seed_from_file('cdo-nces', "2020-2021/pss/schools_private.csv") do |filename|
+        merge_from_csv(filename, {headers: true, encoding: 'ISO-8859-1:UTF-8'}, true, is_dry_run: false) do |row|
```

Comment:
do you want this `is_dry_run` to be true for now?

Reply:
I will update this before merging!